src/main/kotlin/org/opensearch/commons/alerting/action/IndexMonitorRequest.kt

```diff
@@ -16,21 +16,24 @@ class IndexMonitorRequest : ActionRequest {
     val refreshPolicy: WriteRequest.RefreshPolicy
     val method: RestRequest.Method
     var monitor: Monitor
+    val rbacRoles: List<String>?
 
     constructor(
         monitorId: String,
         seqNo: Long,
         primaryTerm: Long,
         refreshPolicy: WriteRequest.RefreshPolicy,
         method: RestRequest.Method,
-        monitor: Monitor
+        monitor: Monitor,
+        rbacRoles: List<String>? = null
```

Comment:
In case of `rbacRoles` set as Null, does this default back to Admin role?

Reply:
This is for BWC, the logic is [here](https://github.com/opensearch-project/alerting/pull/635/files). If null, it would assume no backend role is passed in and will copy the backend roles of the user for monitor creation. If it was updating a monitor, the backend roles of the monitor wont change.